packages/esbuild/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@size-limit/esbuild",
+  "version": "6.0.4",
+  "description": "Esbuild plugin for Size Limit",
```

```suggestion
  "description": "esbuild plugin for Size Limit",
```

Let’s use downcased name. esbuild’s author prefer it.